### PR TITLE
[DOCU-1812]Adds changelog for 2.5.1.0 EE kong gateway release

### PR DESCRIPTION
**Pull request by HCloward**
### Summary
UpdateS changelog with EE Kong Gateway v2.5.1.0 patch release
### Reason
[DOCU-1812](https://konghq.atlassian.net/browse/DOCU-1812)
### Testing
Will include sample build link when available.

### CODE REVIEW

File: app/enterprise/changelog.md [hunk 1]
```diff
@@ -4,8 +4,46 @@ no_search: true
...
+
+#### Plugins
+- [Azure Functions](/hub/kong-inc/azure-functions) (`azure-functions`)
+  This relese updates the `lua-resty-http` dependency to v0.16.1, which means the plugin no longer
+  uses the deprecated functions.
+
+
 ## 2.5.0.2
-**Release Date** 2021/09/02
+**Release Date** 2021/09/02cf
```
Comment (mikefero) [inline app/enterprise/changelog.md:46]: is the cf an accident?

**Review by mikefero — APPROVED** [review]
other than the cf question it looks good to me

File: app/enterprise/changelog.md (hunk 1 above)
Comment (HCloward) [reply app/enterprise/changelog.md]: yeah, oops =)

**Review by GayleNeumann — APPROVED** [review]
LGTM

**Pull request merged**